bambi/backend/links.py

```diff
@@ -1,33 +1,37 @@
 import numpy as np
-import aesara.tensor as at
+
+try:
+    import pytensor.tensor as pt
+except ModuleNotFoundError:
```

Comment:
IMO we should just hard cut over. Theres no reason to stay on Aesara and it just means that we may get user bug reports from two backends rather than one.

Reply:
I would recommend to keep compatibility for at least one release and print a warning.

In the four packages I patched in the past few days I maintained compatibility with v4 because it simplifies updating my environments and Docker images.

In any case please take over from here - I'm not even using bambi myself 🙊

Reply:
Ill merge it. Hopefully no one gets angry at me